jemalloc-sys/configure/configure

```diff
@@ -862,6 +863,7 @@ datadir='${datarootdir}'
 sysconfdir='${prefix}/etc'
 sharedstatedir='${prefix}/com'
 localstatedir='${prefix}/var'
+runstatedir='${localstatedir}/run'
```

Comment:
Why the change?

Reply:
I updated the configure file (`cd jemalloc-sys/jemalloc && autoconf && cp configure ../configure/configure`) as a matter of habit when modifying autoconf-based sub modules. We can remove it if you prefer.

Reply:
I think it's related to the version of autoconf. Unless it can fail compilation, better leave them untouch.